tests/integration/test_vulnerability_detector/test_feeds/alas/test_extra_tags_alas_feed.py

```diff
@@ -0,0 +1,110 @@
+import os
+from copy import deepcopy
+
+import pytest
+import wazuh_testing.vulnerability_detector as vd
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.file import truncate_file, read_json_file, write_json_file, write_file
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.services import control_service
+from wazuh_testing.tools.utils import lower_case_key_dictionary_array
+
+# Marks
+pytestmark = [pytest.mark.server, pytest.mark.tier(level=2)]
+
+# Variables
+current_test_path = os.path.dirname(os.path.realpath(__file__))
+test_data_path = os.path.join(current_test_path, '..', '..', 'data')
+configurations_path = os.path.join(test_data_path, 'configuration', 'test_feeds', vd.INVALID_ALAS_FEEDS_CONF)
+custom_alas_json_feed_path = os.path.join(test_data_path, 'feeds', vd.CUSTOM_ALAS_JSON_FEED)
+custom_alas2_json_feed_path = os.path.join(test_data_path, 'feeds', vd.CUSTOM_ALAS2_JSON_FEED)
+
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+# Set Amazon Linux 1 and 2 configuration
+# Set Amazon Linux 1 and 2 configuration
+parameters = [
+    {'ALAS_CUSTOM_FEED': custom_alas_json_feed_path, 'OS': 'amazon-linux'},
+    {'ALAS_CUSTOM_FEED': custom_alas2_json_feed_path, 'OS': 'amazon-linux-2'}
+]
+metadata = lower_case_key_dictionary_array(parameters)
+ids = [
+    'ALAS_configuration',
+    'ALAS2_configuration'
+]
+
+
+# Configuration data
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+
+
+@pytest.fixture(scope='module', params=configurations, ids=ids)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+@pytest.fixture
+def modify_feed(test_values, get_configuration, request):
+    """Modify the Amazon Linux JSON feed by setting a test tag value."""
+    alas_feed_path = get_configuration['metadata']['alas_custom_feed']
+    if alas_feed_path == custom_alas_json_feed_path:
+        custom_feed_path = custom_alas_json_feed_path
+    elif alas_feed_path == custom_alas2_json_feed_path:
+        custom_feed_path = custom_alas2_json_feed_path
+
+    backup_data = read_json_file(custom_feed_path)
+    modified_data = deepcopy(backup_data)
+
+    # Insert key:value pair as string, since otherwise, you could not insert lists or dictionaries as a key
+    modified_string_data = vd.insert_data_json_feed(data=modified_data, field_name=test_values[0],
+                                                    field_value=test_values[1], append_data=None, brackets=False)
+
+    write_file(custom_feed_path, modified_string_data)
+
+    vd.clean_vuln_and_sys_programs_tables()
+
+    control_service('restart', daemon='wazuh-modulesd')
+
+    vd.set_system(system='ALAS')
+
+    yield
+
+    write_json_file(custom_feed_path, backup_data)
+
+    vd.clean_vuln_and_sys_programs_tables()
+
+    truncate_file(LOG_FILE_PATH)
```

Comment:
```suggestion
@pytest.fixture(scope="function")
def modify_feed(test_values, get_configuration, request):
    """Modify the Amazon Linux JSON feed by setting a test tag value."""
    custom_feed_path = get_configuration['metadata']['alas_custom_feed']

    backup_data = read_json_file(custom_feed_path)
    modified_data = deepcopy(backup_data)

    # Insert key:value pair as string, since otherwise, you could not insert lists or dictionaries as a key
    modified_string_data = vd.insert_data_json_feed(data=modified_data, field_name=test_values[0],
                                                    field_value=test_values[1], append_data=None, brackets=False)

    write_file(custom_feed_path, modified_string_data)

    # Clean vuln tables and restart wazuhdb
    vd.clean_vuln_and_sys_programs_tables()

    yield

    write_json_file(custom_feed_path, backup_data)
```

Reply:
Done in commit 330f641b6fd3cb999e7d4142cfe95ec1854f80ef